src/backend/web/validation.js

```diff
@@ -49,5 +50,25 @@ function validateNewFeed() {
     }
   };
 }
+const postQueryValidationRules = [
+  query('per_page', 'per_page needs to be empty or a integer').custom(
+    (value) => !value || Number.isInteger(+value)
```

Comment:
express-validator already has isInt() and exists(), which can be chained and used here, instead of creating a custom validator

Reply:
Can you chain isEmpty() with IsInt ?

Reply:
I dont need exists I need isEmpty() and I do not think you can chain isEmpty and IsInt